README.md

```diff
@@ -249,6 +249,16 @@ It is due to invalid archs, search for them and check it. Maybe you
 targetted a simulator but have only armv7/arm64. Maybe you want to target
 your iPad but it as only x86_64.
 
+### Why does the python multiprocess module/async call not work?
```

Comment:
Why also `async` ? 
Seems that asyncio own subprocess module only imports `PIPE`, `STDOUT`, `DEVNULL` from subprocess (https://github.com/kivy/kivy-ios/issues/372#issuecomment-534253980)
I would change this one to **"Why does the python multiprocess/subprocess module not work?"**

Reply:
Agreed. Done.